crates/forge/src/collecting.rs

```diff
@@ -123,13 +137,23 @@ pub fn filter_tests_from_crates(
     tests_from_crates: Vec<TestsFromCrate>,
     runner_config: &RunnerConfig,
 ) -> Vec<TestsFromCrate> {
-    if let Some(test_name_filter) = &runner_config.test_name_filter {
+    let filtered_by_name = if let Some(test_name_filter) = &runner_config.test_name_filter {
         tests_from_crates
             .into_iter()
             .map(|tc| tc.filter_by_name(test_name_filter, runner_config.exact_match))
             .collect()
     } else {
         tests_from_crates
+    };
+
+    // TODO: add tests for filter_tests_from_crate
```

Comment:
Will do on Friday

Reply:
Actually it is tested in e2e tests, the function filtering ignore is so simple that I won't unit test it